Sync Properties Update Revision Block and Create PDF/Modules/Sample.SyncFilePropsAndUpdRevBlockModule.psm1

```diff
@@ -824,7 +824,8 @@ function UpdateRevisionBlock {
 	param ($file)
 
 	#using Vault API to get the file instead of using powerVault's $file since it runs into property sync job not finding any equivalence problems when '$file.Name' or '$file._Name' is used
-	$fileForUpdRevJob = $vault.DocumentService.GetLatestFileByMasterId($file.MasterId)
+	#$fileForUpdRevJob = $vault.DocumentService.GetLatestFileByMasterId($file.MasterId)
+	$fileForUpdRevJob = $file
```

Comment:
instead of assigning the $file to local variable, rename the param $file to $fileForUpdRevJob. 
remove the comment also since it is not relevant.

Reply:
are you sure that this is not needed anymore or do I need to test it?

Reply:
right. Good to do a test. 
the comment is old and maybe not relevant.  so that can be removed.
but if the update job requires latest file then we need to keep the call to get the latest file.